portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -47,10 +51,18 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     response.setContentType("text/html;");
     response.getWriter().println(name + ": " + comment);
 
+    // Perform sentiment analysis.
```

Comment:
👍 Good design to do this once when the comment is written rather than each time a comment is read out from the Datastore.